### DISCUSSION TITLE
Indentation in summaries

### DISCUSSION THREAD

**Discussion by jotaen** [jotaen/klog#49] [Feature ideas & discussions]
Say you want to nest bullet points:

```
- Something
  - Note 1
  - Note 2
```

That’s currently not possible in record summaries, because the indentation is reserved for record entries. A workaround would be to prefix the summary with a character (`|`, `#`, etc.):

```
2020-01-01
| - Something
|   - Note 1
|   - Note 2
  1h
```

That solves the indentation problem, but then the `|` characters would be part of the copy.

I’m not sure, however, whether this is necessary at all?

**Comment by vladdeSV** [comment]
Is this limited to summaries directly after dates, or for entries as well?

**Reply by jotaen** [reply]
I’d say it’s only needed in the record summary (the one below the date), because should the entry summary support multiple lines then you can increase the indentation anyways:

```
2020-01-01
    1h Did foo
         - Note 1
           - Note 2
    2h Something
```

**Reply by jotaen** [reply]
Aside: I added a remark in the description above, because I’m not sure at the moment whether this “escaping” feature would bring enough benefit at all, so I’d put it on hold for now. 🤔

**Reply by vladdeSV** [reply]
In general, I feel as there are many larger changes proposed right now. Take your time on deciding on any other larger change.

If you really like a feature, implement it on a separate branch and toy around with it, then use it for a week. In my opinion, that is the best way to see if a feature is necessary or not.
